docs/disaster-recovery.md

```diff
@@ -0,0 +1,399 @@
+# Failover and Failback In DR
+
+[RBD mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/) is an asynchronous replication of RBD images between multiple Ceph clusters.
+This capability is available in two modes:
+
+* Journal-based: Every write to the RBD image is first recorded to the associated journal before modifying the actual image. The remote cluster will read from this associated journal and replay the updates to its local image.
+* Snapshot-based: This mode uses periodically scheduled or manually created RBD image mirror-snapshots to replicate crash-consistent RBD images between clusters.
+
+Here, Snapshot based mirroring approach is used to asynchronously replicate RBD images for Disaster Recovery.
+
+## Create Ceph Pools
+
+In this section we create specific ceph pools that are RBD mirroring enabled for use with the DR use case.
+
+>:bulb: **Note:** It is also feasible to edit existing pools and enable them for replication.
+
+Execute the following steps on each peer cluster to create mirror enabled pools:
+
+* Create a Ceph pool that is mirroring enabled, adding the section spec.mirroring in the CephBlockPool CR, example:
+
+    ```yaml=1
+    apiVersion: ceph.rook.io/v1
+    kind: CephBlockPool
+    metadata:
+      name: replicapool
+      namespace: rook-ceph
+    spec:
+      replicated:
+        size: 1
+      mirroring:
+        enabled: true
+        mode: image
+        # schedule(s) of snapshot
+        snapshotSchedules:
+          - interval: 24h # daily snapshots
+            startTime: 14:00:00-05:00
+    ```
+
+* Creating replicapool pool on the cluster:
+
+    ```bash
+    kubectl apply -f pool.yaml -n rook-ceph
+    ```
+
+    > ```bash
+    > cephblockpool.ceph.rook.io/replicapool created
+    >
+
+* Repeat the steps on the peer cluster.
+
+>:memo: WARNING: Pool name across the cluster peers should be the same for RBD replication to function.
+
+For more information on CephBlockPool CR, please refer to the [documentation](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md#mirroring).
+
+## Bootstrap Peers
+
+In order for the rbd-mirror daemon to discover its peer cluster, the peer must be registered and a user account must be created. The following steps enable bootstrapping peers to discover and authenticate to each other.
+
+For more details, refer to the official rbd mirror documentation on [how to create a bootstrap peer](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#bootstrap-peers).
+
+## Create RBDMirror CRD
+
+Replication is handled by the rbd-mirror daemon. The rbd-mirror daemon is responsible for pulling image updates from the remote, peer cluster, and applying them to image within the local cluster.
+
+Creation of the rbd-mirror daemon(s) is done through the custom resource definitions (CRDs).
+For more information on how to set up Ceph RBDMirror CRD, refer to [documentation](https://rook.io/docs/rook/master/ceph-rbd-mirror-crd.html).
```

Comment:
this should also point to ceph doc. in cephcsi we should assume that cephcsi is deployed to work with standalone ceph cluster.